inji-web/src/__tests__/components/Common/ErrorSheildIcon.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { ErrorSheildIcon } from '../../../components/Common/ErrorSheildIcon';
+
+describe("Test ErrorSheildIcon Component", () => {
+    test('renders the ErrorSheildIcon component', () => {
+        render(<ErrorSheildIcon />);
```

Comment:
render method can be moved to beforeEach

Reply:
the render method has now been replaced with the render with Provider (function that takes care of rendering)